atomic_reactor/plugins/build_source_container.py

```diff
@@ -37,9 +37,10 @@ def export_image(self, image_output_dir: Path) -> Dict[str, Union[str, int]]:
         source_img = 'oci:{}'.format(image_output_dir)
         dest_img = 'docker-archive:{}'.format(output_path)
         cmd += [source_img, dest_img]
+        cleanup_cmd = ['rm', output_path]
```

Comment:
The output_path is still a `Path`, no? The one that gets formatted into a string is dest_img

Reply:
oh! That's right, we do need to convert it to string.

Reply:
updated